src/core/lender/test/mock/tranche.sol

```diff
@@ -12,43 +11,24 @@
 //
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
-
 pragma solidity >=0.4.24;
-
 contract TrancheMock {
-
-    uint public callsBalance;
-    uint public callsBorrow;
-    uint public callsRepay;
-    uint public callsDebt;
-
-    uint public debtOf; function setDebtOf(uint debt_) public {debtOf=debt_;}
-    uint public balanceOf; function setBalance(uint balance_) public {balanceOf=balance_;}
-
-    uint public supplyRate;
-    uint public rate;
-    uint public reserve;
-    int public loanAmount;
-
+    mapping (bytes32 => uint) calls;
+    mapping (bytes32 => uint) returnValues;
+    function setReturn(bytes32 name, uint returnValue) public {
+        returnValues[name] = returnValue;
+    }
+    function funcBody(bytes32 name) internal returns (uint) {
```

Comment:
👍 this is cool, we should introduce this pattern for all of our mocks

Reply:
sounds good. How about calling the function simply -> call()